touchforms/formplayer/static/formplayer/script/fullform-ui.js

```diff
@@ -1,634 +1,459 @@
-var JST_BASE_DIR = 'formplayer/templates/formplayer/'
 var markdowner = window.markdownit();
 
-function getForm(o) {
```

Comment:
this crashed my computer multiple times when refactoring and messing up the parent relationship. no longer need it though since the only reason we need to get the form was to get the adapter. now we publish events